comet/archiver.py

```diff
@@ -0,0 +1,156 @@
+"""
+Archiver for CoMeT.
+
+Moves comet broker dumps to a database
+"""
+
+import asyncio
+import datetime
+import orjson as json
+import os
+import signal
+
+from time import sleep
+import logging
+
+import chimedb.core as chimedb
+import chimedb.dataset as db
+
+from . import Manager, CometError, __version__
+from .broker import DEFAULT_PORT
+from .manager import TIMESTAMP_FORMAT, LOG_FORMAT
+
+# _STATE_DIR = "/var/lib/comet-archiver"
+
+logging.basicConfig(format=LOG_FORMAT)
+logger = logging.getLogger("comet.archiver")
+logger.setLevel('INFO')
+
+
+class Archiver():
```

Comment:
```suggestion
class Archiver:
```

Reply:
Either things should be based off object explicitly or not have any parentheses.